### PR TITLE
Fix/scanner play

### PR DESCRIPTION
**Pull request by franrull**

### CODE REVIEW

File: src/components/AssetScanner/Webcam/Webcam.tsx [hunk 1]
```diff
@@ -170,7 +167,7 @@ export class Webcam extends Component<WebcamProps, WebcamState> {
     return (
       <StyledVideo
         autoPlay={true}
-        muted={this.props.audio}
+        muted
```
Comment (YevheniiMaltsev) [inline src/components/AssetScanner/Webcam/Webcam.tsx:170]: I don't know the gearbox, but it looks like the previous solution had some mechanism to turn on/off the audio. Should we always disable audio?

Comment (franrull) [reply src/components/AssetScanner/Webcam/Webcam.tsx:170]: Does look like that but I cant think if why we would need the audio on the Webcam component that is only used y the WebCamScanner (and only by Infield)

Comment (franrull) [reply src/components/AssetScanner/Webcam/Webcam.tsx]: @YevheniiMaltsev the underlying problem is that the component should have been muted based on the props.audio passed but the condition was wrong ...  I've changed the condition now, please check again.

Comment (YevheniiMaltsev) [reply src/components/AssetScanner/Webcam/Webcam.tsx]: Yeah, sorry for confuse. I didn't request changes, I just wanted to understand if it's okay 😊

**Review by YevheniiMaltsev — APPROVED** [review]

**Pull request merged**